src/shared/components/templates/Default/Default.tsx

```diff
@@ -1,17 +1,14 @@
+import {Route, RouteComponentProps} from "@shared/components";
 import * as React from "react";
-import { Link, RouteComponentProps } from "react-router-dom";
+import {Link, Switch} from "react-router-dom";
```

Comment:
We should add spaces inside `{}` to the linter, it's a bit inconsistent with some having a space and some not. No preference on which we go with but just so we know what it is.